gcexport.py

```diff
@@ -963,50 +891,28 @@ def fetch_userstats(args):
     return json.loads(result)
 
 
-def extract_display_name(profile_page):
```

Comment:
I guess you'd need to remove also `test_extract_display_name` in gcexport_test.py